ArcadiaTeamsBot/Startup.cs

```diff
@@ -1,13 +1,16 @@
 namespace ArcadiaTeamsBot
 {
     using ArcadiaTeamsBot.Infrastructure;
-
     using Microsoft.AspNetCore.Builder;
     using Microsoft.AspNetCore.Hosting;
     using Microsoft.Bot.Builder;
     using Microsoft.Bot.Builder.Integration.AspNet.Core;
     using Microsoft.Extensions.DependencyInjection;
     using Microsoft.Extensions.Hosting;
+    using MediatR;
+    using ServiceDesk.Abstractions;
+    using ServiceDesk;
+    using ArcadiaTeamsBot.CQRS.Handlers;

```

Comment:
Usings from one publisher should be grouped.

Reply:
Fixed